### PR TITLE
feat(config): add example config for owncloud and nextcloud

### PR DESCRIPTION
**Pull request by hirschenberger**

### CODE REVIEW

File: config/services/webdav.toml [hunk 1]
```diff
@@ -2,8 +2,8 @@
 repository = "opendal:webdav"
 password = "my-backup-password"
 
-# You have to create a passcode on you owncloud instance via
-# Settings -> Security -> App passwords / tokens
+# You have to create a passcode on your cloud instance.
+# * Owncloud: `Settings -> Security -> App passwords / tokens`
```
Comment (simonsan) [inline config/services/webdav.toml]:
```suggestion
# You have to create a passcode on your cloud instance with `Settings -> Security -> App passwords / tokens`.
```

**Review by aawsome — APPROVED** [review]
LGTM!

File: config/services/webdav.toml [hunk 2]
```diff
@@ -2,8 +2,8 @@
...
 [repository.options]
 endpoint = "https://my-owncloud-server.com"
```
Comment (simonsan) [inline config/services/webdav.toml]:
```suggestion
endpoint = "https://my-owncloud-or-nextcloud-server.com"
```

File: config/services/webdav.toml [hunk 3]
```diff
@@ -2,8 +2,8 @@
...
 root = "remote.php/webdav/my-folder"         # This path may be different for your cloudprovider
```
Comment (simonsan) [inline config/services/webdav.toml]:
```suggestion
# root = "remote.php/webdav/my-folder"         # for owncloud
# root = "remote.php/dav/files/<username>"     # for nextcloud
```

**Review by simonsan — CHANGES_REQUESTED** [review]

**Comment by simonsan** [comment]
I'll fix it myself on main ......

**Pull request merged**